src/applications/vaos/appointment-list/redux/reducer.js

```diff
@@ -313,15 +320,18 @@ export default function appointmentsReducer(state = initialState, action) {
         return { ...newAppt, status: APPOINTMENT_STATUS.cancelled };
       });
 
-      let currentAppointment = state.currentAppointment;
+      let appointmentDetails = state.appointmentDetails;
```

Comment:
We had this `currentAppointment` state that seemed unnecessary with `appointmentDetails` and some updated selectors, so I removed it and updated this to make sure we change the currently viewed appointment/request to canceled.